### PR TITLE
Added Git Workflow to README

### PR DESCRIPTION
**Pull request by pirolster**
So it says that my branch can't automatically be merged with the 'develop' branch. I think this is because on my computer I was on main when I pulled before I created my new branch, and the README on 'main' and 'develop' are different. So I feel like we will continue to run into lots of these "can't automatically merge" messages if we continue to pull from 'main' but then do work that we will ultimately want to merge into 'develop.' What do people think? 

Subsequently let me know if anything about this git workflow needs to change and I will fix it today.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,26 @@
...
+     * You should see the command-line prompt �Switched to a new branch�
+     * You can use the command: �git log --all --decorate --oneline --graph� to display the entire DAG and see where you are
+     * You can see just the branch you�re on with the command: �git branch --show-current� to confirm you�re in the right place
+4. Do your business on your branch. Right now this is only on your local machine, so do what you want. You won�t mess anything up for anyone else and no one can judge you for any dum dum things you want to test out� not that anyone would anyway ;)
+5. When you are ready to send your work upstream�
+     * Stage your files with �git add your_filename�
+     * Commit with �git commit -m �your commit log� �
+     * Do your best to make your commit log legible to more than just you :)
+     * Finally push your files with �git push origin your_name-name_of_feature (i.e. the name of the branch you created for your work)�
+6. Your push will create a pull request. If you go to the github repository in your browser, you will see this under the �Pull request� tab (between �Issues� and �Actions�)
```
Comment (JordanHamel) [inline README.md:19]: Just pushing your branch to github will not create a pull request. You need to do that manually on github.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,26 @@
...
+     * If you click on the pull request you just created, you should be able to see the commit history, changelogs of all the files you pushed, and a text box that allows you to leave a comment. 
+     * On the right side of the webpage, you will see �Assignees� (between �Reviewers� and �Labels�). Assign at least 2 other people to the pull request. These teammates will be responsible for reviewing your code and ultimately merging the changes into �develop�. Try to make fairly regular pushes, so that the code review for your teammates is more manageable. They will ultimately decide if it is okay to be merged into the development branch. There should be no reason for command-line level merges
+     * Once you�ve completed Step 5b, click the green button �Create Pull Request�
+7. When you want to update the files on your local machine, for example, to retrieve the work someone else has pushed to the repository
+     * make sure you first switch to main with �git checkout main�
+     * Type �git pull origin main�
+
```
Comment (JordanHamel) [inline README.md:26]: Need to add something about the workflow for merging develop into main

**Comment by JordanHamel** [comment]
Re: merge conflicts

For this time, the conflicts can be manually resolved using the "resolve conficts" button above. Do this once we're ready to merge (after reviewer comments have been resolved).

For next time, Gabe is right that the best strategy is to avoid working on a file at the same time as somebody else. Once we get more comfortable with our git workflow, things will get merged into main more quickly and this is less likely to happen.

**Comment by pirolster** [comment]
Closing this because I'll make a new pull request when I've addressed Jordan's comments

**Pull request closed without merging**